Story/0AllStories.cs

```diff
@@ -295,7 +298,6 @@ public class AllStories
     public ExaltiaTower ExaltiaTower = new();
 
     public FrozenNorthlands FrozenNorthlands = new();
-    public FableForest FableForest = new();
```

Comment:
apparently you forced push your pull request before updating your scripts so it deleted a recent update on 0Allstories